src/main/java/uk/gov/ons/census/casesvc/service/CaseService.java

```diff
@@ -27,6 +27,7 @@
   private static final Logger log = LoggerFactory.getLogger(CaseService.class);
   private static final String CASE_NOT_FOUND_ERROR = "Case not found error";
   private static final String SURVEY = "CENSUS";
+  private static final String HOUSE_RESPONSE_ADDRESS_TYPE = "HH";
```

Comment:
I guess it doesn't really matter, but maybe we should use "household" instead of "house" given that's the terminology used everywhere else

Reply:
Changed.